src/data/reports/patientReport.ts

```diff
@@ -0,0 +1,5 @@
+export interface ReportPatient{
```

Comment:
can u check if you can use the same Patient entity? i mean if we have the same entity with the same data, please double check if we can use Patient and remove this class

Reply:
I change the entity

Reply:
please just check if it's working